.travis.yml

```diff
@@ -47,10 +47,10 @@ script:
 
 deploy:
   provider: pypi
-  user: onefinestay
+  user: mattbennett
```

Comment:
i would be happy to keep ofs until we have an issue, and at that point make a nameko one